docs/metalk8s-ui.rst

```diff
@@ -0,0 +1,49 @@
+Metalk8s UI
+===========
+
+
+
+
+.. todo::
+
+  The following test has been realized on vagrant only - One should list
+  all requirements for reaching the UI using a operationnal cluster.
```

Comment:
s/a operationnal/an operationnal/?

Reply:
```suggestion
  all requirements for reaching the UI using an operational cluster.
```

Reply:
Correction done.
Will be effective after the next commit.
THX